--- efivar-0.15/src/makeguids.c~	2014-10-15 15:48:49.000000000 +0200
+++ efivar-0.15/src/makeguids.c	2015-02-23 06:18:10.638509940 +0100
@@ -100,7 +100,7 @@ main(int argc, char *argv[])
 		err(1, "makeguids: could not read \"%s\"", argv[1]);
 
 	/* strictly speaking, this *has* to be too large. */
-	struct guidname *outbuf = calloc(inlen, sizeof (char));
+	struct guidname *outbuf = calloc(inlen*10, sizeof (char));
 	if (!outbuf)
 		err(1, "makeguids");
 
